Source/DafnyCore/Verifier/ProofDependencyManager.cs

```diff
@@ -83,5 +83,9 @@ public class ProofDependencyManager {
         throw new ArgumentException($"Malformed dependency ID: {component.SolverLabel}");
       }
     }
+
+    public IEnumerable<ProofDependency> GetOrderedFullDependencies(IEnumerable<TrackedNodeComponent> components) {
+      return components.Select(GetFullIdDependency).OrderBy(dep => (dep.RangeString(), dep.Description));
```

Comment:
I find ordering by RangeString to be strange. Why not order on the unprinted object: `dep.Range.StartToken` or `dep.Range` ?

Also, Instead of the tuple, you can also use the C# chaining ordering API, `OrderBy(dep => dep.Range.StartToken).ThenBy(dep => dep.Description)`, which I think is a little easier to read.